### DISCUSSION TITLE
State of @UnstableApi

### DISCUSSION THREAD

**Discussion by marschwar** [detekt/detekt#4653] [Ideas]
There are two questions around the `@UnstableApi` annotation I like to discuss.

1. The reporting extension has been an unstable api for 2 years now. Is it still subject to change or could we remove the marker?
2. The fact that the annotation itself is marked with `@RequiresOptIn` makes it a little cumbersome to use. Could we maybe remove the annotation or create an additional annotation such as `org.gradle.api.Incubating`.

**Comment by BraisGabin** [comment]
I think that we should remove those. I mean, they aren't Unstable. We are not going to change them until 2.0 so they should be "safe to use".
